brut.j.util/src/main/java/brut/util/BrutIO.java

```diff
@@ -30,10 +30,8 @@ public static void copyAndClose(InputStream in, OutputStream out)
         try {
             IOUtils.copy(in, out);
         } finally {
-            try {
```

Comment:
Is there any benefit to using this, or just code cleanup?

I see `closeQuietly` does exactly the same thing. Closing the streams ignoring exceptions.